app/controllers/content_management/articles_controller.rb

```diff
@@ -41,6 +41,11 @@ def update
         
     end
 
+    def destroy
+        @article = Article.find(params[:id])
```

Comment:
there is no reason to use an instance variable here

Reply:
We've updated it.  Changed it to local variable.